src/auth/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import {
+    Injectable,
+    UnauthorizedException,
+    InternalServerErrorException,
+} from '@nestjs/common';
+
+@Injectable()
+export class AuthService {
+    getUserInfo(userId: string): Promise<string> {
+        // fetch user info from the database
+        console.log(`fetching user info for userId: ${userId}`);
+        return Promise.resolve(userId); // return userId
+    }
```

Comment:
:wrench: This would usually go into a `UserService` Class since getting a user by id accesses the User Repository and might be needed by more Controllers or Services than just the AuthService. Therefore it could be safely deleted for now :+1: 
```suggestion
    getUserInfo(userId: string): Promise<string> {
        // fetch user info from the database
        console.log(`fetching user info for userId: ${userId}`);
        return Promise.resolve(userId); // return userId
    }
```